_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
```

Comment:
Perhaps we should avoid plural forms. I ran into that problem in database schema naming, so I generally avoid it as a precaution.